packages/@glimmer/syntax/lib/parser/tokenizer-event-handlers.ts

```diff
@@ -361,7 +362,8 @@ const syntax: Syntax = {
 };
 
 export function preprocess(html: string, options?: PreprocessOptions): AST.Program {
-  let ast = typeof html === 'object' ? html : handlebars.parse(html);
+  const ignoreStandalone = options ? options.ignoreStandalone : false;
+  let ast = typeof html === 'object' ? html : handlebars.parse(html, { ignoreStandalone });
```

Comment:
Are there other options that are allowed for `handlebars.parse`? 

Should we try something like this instead:

```ts
export function preprocess(html: string, options?: PreprocessOptions): AST.Program {
  let parseOptions = options ? options.parseOptions : undefined;
  let ast = typeof html === 'object' ? html : handlebars.parse(html, parseOptions);
  // ...snip...
}
```